tests/05-cni.sh

```diff
@@ -24,7 +24,7 @@ function run_cni_container {
 		shift
 	done
 
-	contid=$(docker run -d --net=none $LABELS busybox:latest /bin/sleep 10000000)
+	contid=$(docker run -t -d --net=none $LABELS busybox:latest)
```

Comment:
This looks wrong. The sleep here ensures that the container keeps running, otherwise it will exit immediately.